src/main/webapp/app/shared/pages/privacy-page/privacy-page.component.scss

```
@@ -0,0 +1,54 @@
+.privacy-page {
+  color: var(--p-text-primary);
+  padding: 3rem 5rem 1.5rem 1.5rem;
```

Please adjust the padding here as well
```suggestion
  padding: 3rem 5rem 1.5rem 5rem;
```